.github/workflows/pytest.yml

```diff
@@ -57,4 +57,4 @@ jobs:
           pip install pytest pytest-xdist pytest-env>=0.6 memory-profiler==0.61.0
 
       - name: Run tests with pytest
-        run: SKYPILOT_DISABLE_USAGE_COLLECTION=1 SKYPILOT_SKIP_CLOUD_IDENTITY_CHECK=1 pytest -n 1 --dist no ${{ matrix.test-path }}
+        run: SKYPILOT_DISABLE_USAGE_COLLECTION=1 SKYPILOT_SKIP_CLOUD_IDENTITY_CHECK=1 pytest -n 0 --dist no ${{ matrix.test-path }}
```

Comment:
xdist can shrink a lot messages that might be useful. Also, we have experienced issue with running tests with multiple process in GH action, probably because there are very limited resources with GH action.